### ISSUE TITLE
best_model.pt

### ISSUE THREAD

**Issue by KeyBoarder8** [boathit/research-papers#2]
Because of my poor device or version of pytorch,it shows "CUDA error: out of memory" .Could you please share me the flie “best_model.pt” you get?Google cloud link or other may be OK.Thanks so much!

**Comment by boathit** [comment]
The minimal GPU memory is 6GB. As the work has been done one year ago, I have cleaned up all the intermediate files including `best_model.pt` in the server. I will manage to train a model for you in my machine but it takes time.

**Comment by KeyBoarder8** [comment]
@boathit My GPU memory is just 4GB......Sorry to take your time and I will look forward to your reply.

**Comment by boathit** [comment]
@KeyBoarder8, our trained model is available [here](https://drive.google.com/open?id=1uxZUmvFHhpY8tOXvCDHuEd7KFOTYp109). It was tested with PyTorch 0.1.2 and cuda8.0.

**Comment by KeyBoarder8** [comment]
@boathit Thank you for your kindness!

**Comment by KeyBoarder8** [comment]
@boathit Excuse me,where can I find the skip gram model in your code

**Comment by boathit** [comment]
The main benefit of pre-training is accelerating the subsequent training process. But the training speed actually is not a big problem if you have a strong GPU so I remove it in this repository. If you want to add it, just call the [`load_pretrained_embedding`](https://github.com/boathit/research-papers/blob/master/t2vec/models.py#L176) function at [here](https://github.com/boathit/research-papers/blob/master/t2vec/train.py#L208) by referring the [official tutorial](https://pytorch.org/tutorials/beginner/nlp/word_embeddings_tutorial.html).

**Comment by KeyBoarder8** [comment]
@boathit OK,Thanks!

**Comment by KeyBoarder8** [comment]
@boathit Sorry to bother you again.If I currently have a small number of trajectory data and the region is different from the trajectory data set you use, do I need to retrain the neural network model?

**Comment by boathit** [comment]
Yes, the testing data is supposed to own the same vocabulary as the training data.
